packages/frontend/src/retro/components/buttons/ToggleTimerDialogButton.tsx

```diff
@@ -18,13 +19,19 @@ export function ToggleTimerDialogButton() {
   const { user } = useUserContext();
 
   const { isEffectActive, startEffect } = useTimedEffect({ effectLength: 3000 });
-  const { minutes, seconds, remainingTimeLabel } = useTimer({ onTimerFinish: startEffect });
+  const { minutes, seconds, remainingTimeLabel } = useTimer({ onTimerFinish: handleTimerExpired });
+  const [playTimeExpiredSound] = useSound("/sfx/timer_expired.wav");
 
   function handleOpenDialog() {
     if (!isModerator(user)) return;
     openDialog();
   }
 
+  function handleTimerExpired() {
+    playTimeExpiredSound();
+    startEffect();
+  }
```

Comment:
This function was added but it's not called anywhere or am I missing something?

Reply:
It's used as the onTimerFinish callback in line 22. As soon as the Timer is set to STOPPED, the sound is played (and the button starts to wiggle).